src/pyhf/utils.py

```diff
@@ -67,3 +68,46 @@ def convert(self, value, param, ctx):
             self.fail(
                 '{0:s} is not a valid equal-delimited string'.format(value), param, ctx
             )
+
+
+def digest(obj, algorithm='sha256'):
+    """
+    Get the digest for the provided object. Note: object must be JSON-serializable.
+
+    The hashing algorithms supported are in :module:`hashlib`, part of Python's Standard Libraries.
```

Comment:
`module:` isn't getting rendered correctly on Read The Docs. :? https://pyhf.readthedocs.io/en/feat-workspacehash/_generated/pyhf.utils.digest.html We can debug this tomorrow.

Reply:
`:module:` -> `:mod:`